go.sum

```diff
@@ -112,8 +112,8 @@ github.com/signal-golang/libphonenumber v1.2.2-0.20220127214340-b765372d3707 h1:
 github.com/signal-golang/libphonenumber v1.2.2-0.20220127214340-b765372d3707/go.mod h1:oW+JXk+4C0R5h8BQzfRA4k/vXuBfBcNSqKWjn7Dgncs=
 github.com/signal-golang/mimemagic v0.0.0-20200821045537-3f613cf2cd3f h1:S17lCk5rcobvgcaLf7XSAoLuNkRPgFyhvgRA9JwDzHc=
 github.com/signal-golang/mimemagic v0.0.0-20200821045537-3f613cf2cd3f/go.mod h1:tU6SWwv50oGkZNPlvTFvmqZvEBp0vWWCC+LCEVlTE5A=
-github.com/signal-golang/textsecure v1.13.0 h1:QUkP5ifMUKuxYM4NEzUmtkKLm39G5uykYswNnrzzJwM=
-github.com/signal-golang/textsecure v1.13.0/go.mod h1:sna5yQxNwY/F54q/K4xxP90VgWMSBgm3L8zbWOkF2mk=
+github.com/signal-golang/textsecure v1.16.1-0.20220903213930-c29016f26788 h1:7PvI35zTptU9+8QANzyvx4yHKag6BLUtyW1xrhtKTIE=
+github.com/signal-golang/textsecure v1.16.1-0.20220903213930-c29016f26788/go.mod h1:sna5yQxNwY/F54q/K4xxP90VgWMSBgm3L8zbWOkF2mk=
```

Comment:
No change in _h1_ of _go.mod_ in this case?

Reply:
This is automagically generated, so don't know why.